fluentd/configs.d/filter/k8s_record_transform.conf

```diff
@@ -3,7 +3,7 @@
   enable_ruby
   <record>
     hostname ${(kubernetes_host rescue nil) || File.open('/etc/docker-hostname') { |f| f.readline }.rstrip}
-    message ${log}
+    message ${(message rescue nil) || log}
     version 1.1.4
   </record>
   remove_keys log,stream
```

Comment:
You could probably do it with some ruby - the hard part would be the remove_keys part - you could set log to '' or nil if you don't need it, and see what happens when the elasticsearch plugin formats it